ambry-api/src/main/java/com.github.ambry/router/RouterErrorCode.java

```diff
@@ -82,7 +82,11 @@
   /**
    * TTL of Blob has expired and so Blob cannot be retrieved.
    */
-  BlobExpired;
+  BlobExpired,
+  /**
+   * The range offsets provided for a getBlob operation go past the total size of a blob.
```

Comment:
Is this only if the range goes past the total size or for other invalid scenarios as well?